src/Application.java

```diff
@@ -0,0 +1,58 @@
+import sun.java2d.loops.TransformBlit;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+public class Application {
+
+    public void start() {
+        Transaction[] transactions = getData();
+        Transaction[] orderedTransactions = orderTransactions(transactions);
+        for (Transaction transaction : orderedTransactions) {
+            print(transaction);
+        }
+    }
+
+    public Transaction[] getData() {
+        return new TransactionData().getTransactionData();
+    }
+
+    public Transaction[] orderTransactions(Transaction[] unorderedTransactions) {
+        List<Transaction> transactionList = Arrays.asList(unorderedTransactions);
```

Comment:
I see a lot of new concepts here. You are going to have to walk me through exactly how much of this you are confident in using, just so I know we're not rushing through things at an uncomfortable pace.